images/Dockerfile

```diff
@@ -46,4 +46,8 @@ COPY --chown=runner:docker --from=build /actions-runner .
 
 RUN install -o root -g root -m 755 docker/* /usr/bin/ && rm -rf docker
 
-USER runner
\ No newline at end of file
+USER runner
+
+COPY entrypoint.sh .
+
+ENTRYPOINT [ "./entrypoint.sh" ]
```

Comment:
I guess we have to create an entry point to reload the cert, how should we change the helm chart?

Reply:
You don't need to provide command now by default :relaxed: I tried running it with env JIT and the JIT flag, works fine
So however you want to pass the JIT, it will work
And when you copy assets for example with init containers, entrypoint is in extracted runner directory and it goes with all other files

Reply:
Do we need to add trap sigint sigterm to the entrypoint?

Reply:
No, because we use exec which will replace the entrypoint with the run.sh that already traps the signal :smile: